gulpfile.js

```diff
@@ -408,7 +408,7 @@ const copyTypescript = ((cache) =>
       .src(globs.typescript)
       .pipe(
         gulpFilter([
-          'packages/neo-one-smart-contract-lib/src/*.ts',
+          'packages/neo-one-smart-contract-lib/src/**/*.ts',
```

Comment:
You should maybe split this change into a separate PR as I think it will affect more than just this one. @davemneo

Reply:
I applied it to affected PR's so I don't have to wait for the PR and CI process.